docs/source/getting-started/content/ref_minimum-system-requirements.adoc

```diff
@@ -0,0 +1,14 @@
+[id="minimum-system-requirements_{context}"]
+= Minimum system requirements
+
+{prod} requires the following system resources in order to run OpenShift:
+
+* 4 virtual CPUs (vCPUs)
```

Comment:
Right, we do not have to go into more details, like real cores or hyperthreaded, etc.